packages/@aws-cdk/aws-rds/lib/secret-rotation.ts

```diff
@@ -2,17 +2,26 @@ import ec2 = require('@aws-cdk/aws-ec2');
 import lambda = require('@aws-cdk/aws-lambda');
 import serverless = require('@aws-cdk/aws-sam');
 import secretsmanager = require('@aws-cdk/aws-secretsmanager');
-import cdk = require('@aws-cdk/cdk');
+import { Construct } from '@aws-cdk/cdk';
 
 /**
- * A serverless application location.
+ * A secret rotation serverless application.
  */
-export class ServerlessApplicationLocation {
-  public static readonly MariaDbRotationSingleUser = new ServerlessApplicationLocation('SecretsManagerRDSMariaDBRotationSingleUser', '1.0.46');
-  public static readonly MysqlRotationSingleUser = new ServerlessApplicationLocation('SecretsManagerRDSMySQLRotationSingleUser', '1.0.74');
-  public static readonly OracleRotationSingleUser = new ServerlessApplicationLocation('SecretsManagerRDSOracleRotationSingleUser', '1.0.45');
-  public static readonly PostgresRotationSingleUser = new ServerlessApplicationLocation('SecretsManagerRDSPostgreSQLRotationSingleUser', '1.0.75');
-  public static readonly SqlServerRotationSingleUser = new ServerlessApplicationLocation('SecretsManagerRDSSQLServerRotationSingleUser', '1.0.74');
+export class SecretRotationApplication {
+  public static readonly MariaDbRotationSingleUser = new SecretRotationApplication('SecretsManagerRDSMariaDBRotationSingleUser', '1.0.57');
```

Comment:
This is bumping the version numbers... Should I be afraid of this breaking anyone's stuff?

This also begs the question of how do we keep up to date on those version numbers (but this is not this PR's job to address that!)

Reply:
It's not breaking, while I was in there I updated the versions (the original versions come from #2052).